src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Reload/ProjectReloadManager.cs

```diff
@@ -329,6 +335,7 @@ private void ProcessProjectWhichFailedReload(IReloadableProject project, out boo
                                         VSResources.ReloadAll,
                                         VSResources.Reload};
 
+            // TODO: Consider showing different messages for reload failed and needs forced reload.
```

Comment:
It would be good to show a different reload message here. @nguerrera are we completely locked for any loc changes?